### ISSUE TITLE
Server GUI "forgets" HTML / CSS

### ISSUE THREAD

**Issue by kjcole** [jamulussoftware/jamulus#3050]
**Description**

On Pop! OS 22.04 LTS, running Jamulus 3.9.1, HTML and CSS entered into the GUI form is stripped away, leaving only the plain-text remembered.

**To Reproduce**

1. Start **Jamulus (server)** from the Applications menu.
2. Enter HTML/CSS in the **Chat Window Welcome** textbox.
3. Start Jamulus as a client and connect to the server.
4. HTML and CSS are rendered correctly.
5. Hide the server window (or exit)
6. Show the server window (or restart)
7. HTML / CSS source disappears from the **Chat Window Welcome** textbox, and connecting to the server shows the welcome message in plain-text stripped of all the formatting and styling.

**Expected behavior**

The **Chat Window Welcome** should show the HTML and connecting to the server from the client should render the HTML.

**Operating system**

* Pop! OS 22.04 LTS
* Jamulus 3.9.1 

**Additional context**

( The package is `jamulus_3.9.1_ubuntu_amd64.deb` downloaded from https://jamulus.io/wiki/Installation-for-Linux)

**Comment by pljones** [comment]
@kjcole (once the Assets are built), could you take a look to see if this fixes the problem you were seeing?

**Comment by mcfnord** [comment]
I can't decide if I've noticed a pattern related to this.

I use **-w** or **--welcomemessage**  with a filename at CLI like this:

`jamulus-headless --nogui --server --directoryserver anygenre1.jamulus.io:22124 --serverinfo "Hear;California;US" -u 6  -w /root/welc.html`

That shows the welc.html file contents when I connect.

But then I run jamulus-headless using systemctl, starting with this:

`sudo systemctl edit --full jamulus-headless`

I made this line:

`ExecStart=/bin/sh -c 'exec /usr/bin/jamulus-headless --nogui --server --directoryserver anygenre1.jamulus.io:22124 --serverinfo "Hear;California;US" -u 6  -w /root/welc.html'`

When I connect to that server, I see this:

Server Welcome Message: /root/welc.html

Oh no, that's the path, not the file contents. Is it possible to specify a full path here, and see the file contents? The syntax from CLI to systemctl sure looks the same to me.

(And could the file contents be re-loaded each time it's requested, rather than cached? I'd settle for `systemctl reload jamulus-headless` to re-cache, except -w with a path appears to be treated literally in systemctl.)
